### PR TITLE
Send all sources to new reducer

### PR DESCRIPTION
**Pull request by jcmorrow**
And disable storing events for now. They have not been super useful, and they can really bog down loading. I was investigating some loading stuff and it turns out mostly what my app was doing was writing down events that I never looked at.

### CODE REVIEW

File: src/ui/reducers/protocolMessages.ts [hunk 1]
```diff
@@ -76,7 +76,10 @@ export const { errorReceived, eventReceived, requestSent, responseReceived } =
 
 export default protocolMessagesSlice.reducer;
 
-export const getProtocolEvents = (state: UIState) => state.protocolMessages.events;
+export const getProtocolEvents = (_state: UIState) => {
+  console.log("protocol events are disabled");
+  return [];
```
Comment (jasonLaster) [inline src/ui/reducers/protocolMessages.ts:81]:
```js
if (! features.logProtocolEvents) {
  console.log("protocol events are disabled");
}
return state.protocolMessages.events || []
}
```

File: src/devtools/client/debugger/src/client/index.ts [hunk 2]
```diff
@@ -36,17 +36,22 @@ async function setupDebugger() {
...
+      sourceInfos.push({
+        type: "generated",
+        data: prepareSourcePayload({
+          actor: sourceId,
+          url,
+          sourceMapURL,
+        }),
+      });
+    }
   });
```
Comment (jasonLaster) [inline src/devtools/client/debugger/src/client/index.ts:55]: i can't wait to kill this old code

**Review by jasonLaster — APPROVED** [review]

File: src/devtools/client/debugger/src/client/index.ts (hunk 2 above)
Comment (jcmorrow) [reply src/devtools/client/debugger/src/client/index.ts:55]: Agreed.

File: src/ui/actions/session.ts [hunk 3]
```diff
@@ -167,10 +167,10 @@ export function createSocket(
...
 
       const sessionId = await createSession(recordingId, loadPoint, experimentalSettings, {
-        onEvent: (event: ProtocolEvent) => {
-          if (features.logProtocol) {
-            dispatch(eventReceived({ ...event, recordedAt: window.performance.now() }));
-          }
+        onEvent: (_event: ProtocolEvent) => {
+          // if (features.logProtocol) {
+          //  dispatch(eventReceived({ ...event, recordedAt: window.performance.now() }));
+          // }
```
[the remark this answers is not in the document]
Comment (jcmorrow) [reply src/ui/actions/session.ts:173]: I'm down for that as well.

**Pull request merged**